packages/enzyme-matchers/src/assertions/toIncludeText.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree. *
+ *
+ * @providesModule toIncludeTextAssertion
+ * @flow
+ */
+
+import negateMessage from '../negateMessage';
+import type { Matcher } from '../../../../types/Matcher';
+import type { EnzymeObject } from '../../../../types/EnzymeObject';
+
+export default function toIncludeText(enzymeWrapper: EnzymeObject, text: ?string): Matcher {
+  const actualText = enzymeWrapper.text();
+  let pass;
+
+  if (text === undefined) {
+    pass = actualText.length > 0;
+    return {
+      pass,
+      message: negateMessage(
+        pass,
+        'Expected node to include text'
+      ),
```

Comment:
I feel like with this matcher, it should throw if there is no text passed in. What do you think? My thoughts are that since this is loose, it should define what it wants. Whereas with the exact matcher (`toHaveText`) passing in null is explicit and obvious. 

If not, I would expect someone to directly pass in an empty string to at least comply with the API and make a logical looking test.